lib/reek/smells/too_many_instance_variables.rb

```diff
@@ -35,14 +35,29 @@ def self.default_config
       #
       def sniff(ctx)
         max_allowed_ivars = value(MAX_ALLOWED_IVARS_KEY, ctx)
-        count = ctx.local_nodes(:ivasgn).map { |ivasgn| ivasgn.children.first }.uniq.length
+
+        count = count_instance_variables(ctx)
+
```

Comment:
How about not having those newlines here and above? For me they are not increasing readability but rather decreasing it.